pango/src/auto/enums.rs

```diff
@@ -1004,6 +1010,8 @@ pub enum FontScale {
     Superscript,
     #[doc(alias = "PANGO_FONT_SCALE_SUBSCRIPT")]
     Subscript,
+    #[doc(alias = "PANGO_FONT_SCALE_SMALL_CAPS")]
+    SmallCaps,
```

Comment:
This addition is fine because the enum itself was added in 1.50

Reply:
Adding variants to enums is also generally not a problem

Reply:
my comment was about not having a cfg guard for this but yeah